.github/workflows/flutter_package.yml

```diff
@@ -43,6 +43,10 @@ on:
         required: false
         type: boolean
         default: false
+      very_good_cli_version:
```

Comment:
See my comment on https://github.com/VeryGoodOpenSource/very_good_workflows/issues/90#issuecomment-1440279766 about not tightly coupling the public API of the workflow to very_good_cli since very_good_cli is just an implementation detail imo. I think we should instead just restrict the workflow to a specific version range `dart pub global activate very_good_cli <0.2.0`